src/client/visualizations/time-series/circumstances.tsx

```diff
@@ -0,0 +1,165 @@
+import { $, SortAction } from 'plywood';
+import { List } from 'immutable';
+import { Splits, DataSource, Resolve, SplitCombine, Colors } from '../../../common/models/index';
+import { CircumstancesHandler } from '../../../common/utils/circumstances-handler/circumstances-handler';
+
+export default CircumstancesHandler.EMPTY()
```

Comment:
IMO the statics defined on CircumstanceHandlers are actually circumstances and the circumstances class seems to be doing the circumstance handling?